{% extends "ponsol2web/base_generic.html" %}
{% block content %}
    <!-- Jumbotron -->
    <div class="jumbotron">
        <h1>Wecome to <strong>PON-Sol2</strong></h1>
        <p class="lead">
            Solubility is a central protein property and when reduced can lead to diseases. We have developed a
            prediction method, PON-Sol2, to identify amino acid substitutions that increase, decrease of have no effect
            on the protein solubility. The method is a machine leaning tool utilizing gradient boosting algorithm and it
            has been trained on a large data set of variants with different outcomes after selection of features among
            large number of tested properties. The method is fast and has high performance, which was determined for
            each of the variation types as well as for the entire predictor.
        </p>
        <p><a class="btn btn-lg btn-success" href="{% url "ponsol2:input" %}" role="button">Get started</a></p>
    </div>

    <!-- Example row of columns -->
    <div class="row">
        <div class="col-lg-4 col-md-4">
            <h2>Input Format</h2>
        <div class="col-lg-4 col-md-4">
            <h2>Input formats</h2>
            <p>
                There are two input formats. Sequences can be given either in FASTA format or indicated by their IDs.
            </p>
            <p><a class="btn btn-primary" href="{% url "ponsol2:about" %}#input_example" role="button">View details
                »</a></p>
        </div>
        <div class="col-lg-4 col-md-4">
            <h2>Dataset</h2>
            <p>Altogether 11201 amino acid substitutions in 77 proteins are collected. After filtering, 5666 variants
                are used for training, and 662 variants are used for blind testing. The database is freely available.
                <a href="{% url "ponsol2:download-dataset-ponsol2" %}"> <span class="glyphicon glyphicon-download-alt"></span></a>
        <div class="col-lg-4 col-md-4">
            <h2>Data sets</h2>
            <p>
                5666 variants were used for method training and 662 variants were used for blind test. The data sets are
                freely available.
                <a href="{% url "ponsol2:download-dataset-ponsol2" %}"> <span
                        class="glyphicon glyphicon-download-alt"></span></a>
            </p>
            <p><a class="btn btn-primary" href="{% url "ponsol2:about" %}#dataset" role="button">View details »</a></p>
        </div>
        <div class="col-lg-4 col-md-4">
            <h2>Predict Result</h2>
            <p>
                Result are displayed on the Results page. You can also provide your email, then the results are by
                email.
            </p>
            <p><a class="btn btn-primary" href="{% url "ponsol2:task-list" %}" role="button">View
                details »</a></p>
        </div>
    </div>
    <div class="row text-center" style="margin-top: 50px;">
        <h1 class="text-center text-capitalize">How to cite?</h1>
        <p>
            PON-Sol2 manuscript has been submitted.
            <br>
            {#        <a class="btn btn-success btn-xs" href="#" target="_blank"> Go to Bioinformatics </a>#}
        </p>
        <p class="text-justify">
            {#            PON-Sol (2016), the old version based on Random Forests, has been published Bioinformatics.#}
            {#            <br>#}
            The original PON-Sol was published in
            Yang, Y. , Niroula, A. , Shen, B. , & Vihinen, M. . (2016). Pon-sol: prediction of effects of amino acid
            substitutions on protein solubility. Bioinformatics, btw066.

            <a class="btn btn-success btn-xs"
               href="https://academic.oup.com/bioinformatics/article/32/13/2032/1742792?login=true" target="_blank">
                {#            Go to Bioinformatics#}
                Bioinformatics
            </a>
            {#            Yang Yang, Abhishek Niroula, Bairong Shen, Mauno Vihinen, PON-Sol: prediction of effects of amino acid substitutions on protein solubility, Bioinformatics, Volume 32, Issue 13, 1 July 2016, Pages 2032–2034, https://doi.org/10.1093/bioinformatics/btw066#}
            {#            It was trained this far on the largest dataset of 406 single amino acid substitutions.#}
            {#            It grouped variants into three classes: solubility decreasing and increasing variants and those not affecting solubility.#}
        </p>
        {#        <div class="col-lg-6">#}
        {#            <h2 class="text-center text-capitalize">How to cite?</h2>#}
        {#        </div>#}
        {#        <div class="col-lg-6">#}
        {#            <h2 class="text-center text-capitalize">Dataset</h2>#}
        {#        </div>#}
    </div>
{% endblock %}

